<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rankBrowser</class>
 <widget class="QWidget" name="rankBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="gameRecordTable">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>350</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="globalRankTable">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>350</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="personalInfo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>621</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>620</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
